src/components/RegistrationForm/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import Wrapper from '@components/Wrapper'
+import InputWithLabel from '@components/InputWithLabel'
+import { Form } from './styles'
+
+const RegistrationForm = () => {
+  return (
+    <Wrapper mTop="15px">
```

Comment:
This `mTop="15px"` pattern is a bit weird to me, but why not…

Reply:
The aim of a such pattern is to normalize the margins/paddings values thanks to a single component. 
I'll improve that in order to use values such as `small`, `regular`, `large` who will refer to theme values such as `1em`, `2em`...
What do you think?